src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
```

Comment:
You should initialize ManufacturerDao using provided Injector, as this was explained in the task description:
```suggestion
public class Main {
    private static final String PACKAGE = "src.main.java.mate.jdbc";
    private static final Injector injector = Injector.getInstance(PACKAGE);

    public static void main(String[] args) {
        ManufacturerDao manufacturerDao = (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
```